src/core/core.scale.js

```diff
@@ -384,6 +384,12 @@ class Scale extends Element {
 		me.determineDataLimits();
 		me.afterDataLimits();
 
+		// _configure is called twice, once here, once from core.controller.updateLayout.
```

Comment:
What's the reason for moving this earlier in the process? Does it cause problems in other types of scales?

Reply:
To make `_length` available in `buildTicks`.  `linearBase` is calculating offsets based on ticks there, that are not ready at this point.